[scalability] The asserts for "Average wait for admission" flakes

**What happened**:

The scalability test failed on the assert ` checker_test.go:117: Average wait for admission 9303ms is more then expected 9000ms`:  https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/2070/pull-kueue-test-scheduling-perf-main/1786492813098094592

```
{Failed  === RUN   TestScalability/WorkloadClasses/large
    checker_test.go:117: Average wait for admission 9303ms is more then expected 9000ms
--- FAIL: TestScalability/WorkloadClasses/large (0.00s)
}
```

**What you expected to happen**:

Not to flake.

**How to reproduce it (as minimally and precisely as possible)**:

Repeat the build

**Anything else we need to know?**:

The assert was recently bumped to 9s, but it shows as not enough: https://github.com/kubernetes-sigs/kueue/pull/2067/files.

The estimations [here](https://github.com/kubernetes-sigs/kueue/issues/2066#issuecomment-2079053502) suggest 10s might be needed.

**Environment**:
- Kubernetes version (use `kubectl version`):
- Kueue version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/test/performance/scheduler/default_rangespec.yaml
+++ b/test/performance/scheduler/default_rangespec.yaml
@@ -19,8 +19,8 @@ clusterQueueClassesMinUsage:
   cq: 56 #%
 
 wlClassesMaxAvgTimeToAdmissionMs:
-  # Average value 6666 (+/- 14%), setting at +35%
-  large: 9_000
+  # Average value 6666 (+/- 14%), setting at +35% + 1s accounting for observed failures
+  large: 10_000
 
   # Average value 76768 (+/- 2%), setting at +20%
   medium: 90_000

